examples/recipes/request_id_context.py

```diff
@@ -0,0 +1,19 @@
+# context.py
+
+import threading
```

Comment:
can't you just update this here, instead of having another PR for #2260 ?

Reply:
I don't think you would even need this wrapper class.

Reply:
I didn't want to alter that recipe too much, I'm not even the author. IIRC it was created by various people during PyCon US 2019 Sprints.